### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by fdpierson** [fdpierson/dungeon#1]
We'll communicate through the issues section. If you have any ideas for the project, post them here.

**Comment by DerekLester** [comment]
Awesome. I still like the idea of the mini games. I think they should be random like treasures and enemies would be and if the mini games are completed then you should get some reward, maybe extra health or experience points, something of that nature.

**Comment by fdpierson** [comment]
I would do as much random generation as possible. It's a tried and tested formula used by many RPGs.

**Comment by DerekLester** [comment]
Yeah. Since we haven't decided how you win the game or what the full game is gonna be or even how long the game is I need to think of how many mini games to have.
